### ISSUE TITLE
Error loading assets on a mix release

### ISSUE THREAD

**Issue by maukbe** [elixir-error-tracker/error-tracker#30]
I have added the error_tracker_dashboard at /errors in router. This works fine when run locally using an iex shell. 

Deploying the app to Fly IO it will fail to load the page with the error:
```
Request: GET /errors

[info] ** (exit) an exception was raised:

[info] ** (File.Error) could not read file "/app/_build/prod/lib/error_tracker/priv/static/app.css": no such file or directory
```

Some possibly useful snippets from the Dockerfile used to generate the image being ran:
```
RUN cd assets && npm install 

COPY package.json package.json
RUN npm install

# compile assets
RUN mix assets.deploy

# Compile the release
RUN mix compile

# Changes to config/runtime.exs don't require recompiling the code
COPY config/runtime.exs config/

COPY rel rel
RUN mix release
```
...
```
# Only copy the final release from the build stage
COPY --from=builder --chown=nobody:root /app/_build/${MIX_ENV}/rel/app_name ./

USER nobody

# If using an environment that doesn't automatically reap zombie processes, it is
# advised to add an init process such as tini via `apt-get install`
# above and adding an entrypoint. See https://github.com/krallin/tini for details
# ENTRYPOINT ["/tini", "--"]

CMD ["/app/bin/server"]
```

**Comment by crbelaus** [comment]
Thanks for the report. There seems to be something wrong when using ErrorTracker with a mix releas. Will take a look at it today.

**Comment by stevegrossi** [comment]
Thanks for logging this, @maukbe. As mentioned in the forums I'm having the same issue. However when I run `mix release` to generate a release locally and run it, I do not have this issue and the assets load correctly.

**Comment by crbelaus** [comment]
I've opened a PR that should fix this problem. To test it you need to update your mix.exs to:

```elixir
def deps
  {:error_tracker, "~> 0.1", github: "elixir-error-tracker/error-tracker", branch: "runtime-asset-paths"}
end
```

**Comment by stevegrossi** [comment]
I've deployed the updated version and can confirm it resolves the issue! 🎉 🙇🏻 Appropriately, I was able to resolve the asset error via the ErrorTracker web UI itself 😉 

<img width="1336" alt="Screenshot 2024-08-05 at 1 47 42 PM" src="https://github.com/user-attachments/assets/15b53584-0f83-4f58-acb6-d9a98353b2a7">

**Comment by maukbe** [comment]
Haha that's great - can confirm that fix works for me.

**Comment by crbelaus** [comment]
Great! Will merge it this evening and cut a new release.

**Comment by crbelaus** [comment]
This is now fixed in v0.1.2
